src/Makefile.am

```diff
@@ -314,14 +317,21 @@ libbitcoin_common_a_SOURCES = \
   keystore.cpp \
   netaddress.cpp \
   netbase.cpp \
-  policy/feerate.cpp \
   protocol.cpp \
   scheduler.cpp \
   script/sign.cpp \
   script/standard.cpp \
   warnings.cpp \
   $(BITCOIN_CORE_H)
 
+# policy: shared between all executables that require policy rules.
+libbitcoin_policy_a_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
+libbitcoin_policy_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+libbitcoin_policy_a_SOURCES = \
```

Comment:
It's an in-between step and allows to grow the independent policy code/module. `libbitcoin_policy_a` does only contain the complete independent policy classes. But we could also completely avoid the new "package" (aka library). Thoughts @theuni?